chia/plotters/bladebit.py

```diff
@@ -0,0 +1,199 @@
+import asyncio
+import json
+import traceback
+import subprocess
+import os
+import sys
+import logging
+
+from pathlib import Path
+from typing import Any, Dict, Optional
+from chia.plotting.create_plots import resolve_plot_keys
+from chia.plotters.plotters_util import run_plotter, run_command
+
+log = logging.getLogger(__name__)
+
+
+BLADEBIT_PLOTTER_DIR = "bladebit"
+BLADEBIT_MIN_RAM_GIBS = 416
+
+
+def is_bladebit_supported() -> bool:
+    return sys.platform.startswith("linux") or sys.platform in ["win32", "cygwin"]
+
+
+def meets_memory_requirement(plotters_root_path: Path) -> bool:
+    have_enough_memory: bool = False
+    if get_bladebit_executable_path(plotters_root_path).exists():
+        try:
+            proc = subprocess.run(
+                [os.fspath(get_bladebit_executable_path(plotters_root_path)), "--memory-json"],
+                capture_output=True,
```

Comment:
Or maybe use the existing `run()` wrapper.

```suggestion
                capture_output=True,
                check=True,
```

Reply:
Updated the remaining subprocess.run() calls to use run_command(). Required a few changes to run_command() but operates the same as before.